lib/commands/context.js

```diff
@@ -362,6 +362,26 @@ extensions.onPageChange = async function onPageChange (pageChangeNotification) {
   let newIds = [];
   let newPages = [];
   let keyId = null;
+
+  // Checks if a URL is blacklisted in the 'safariIgnoreWebHostnames' capability
+  function isUrlBlacklisted (url, safariIgnoreWebHostnames) {
```

Comment:
[nits]
`isUrlIgnorelisted`? (just following `safariIgnoreWebHostnames`'s naming)


Could you add a docstring?